### PR TITLE
Byday and byyear-byday buttons padding in recurrence form

### PR DESCRIPTION
**Pull request by sabrina-bongiovanni**
This PR fixes

- the byday field buttons in the recurrence widget when font-size is >16px.

image

- the byyear-byday buttons for the yearly recurrence when font-size is >16px.

image

----
📚 Documentation preview 📚: https://volto--6070.org.readthedocs.build/

### CODE REVIEW

File: packages/volto/theme/themes/pastanaga/extras/widgets.less [hunk 1]
```diff
@@ -153,3 +153,22 @@ body.babel-view .field.language-independent-field {
 .react-select__option {
   .word-break();
 }
+
+// makes sure recurrence form buttons don't overflow when font is big
```
Comment (stevepiercy) [inline packages/volto/theme/themes/pastanaga/extras/widgets.less:157]:
```suggestion
// makes sure event recurrence form buttons don't overflow when font is big
```

File: packages/volto/news/6070.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Byday and byyear-byday buttons padding in recurrence form for big fonts @sabrina-bongiovanni
```
Comment (stevepiercy) [inline packages/volto/news/6070.bugfix:1]:
```suggestion
Remove left and right padding from _Event > Edit recurrence > Repeat on_ buttons when repeating for weekly or yearly events for big fonts, preventing overflow. @sabrina-bongiovanni
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Use a more explicit description from a user perspective

**Review by stevepiercy — APPROVED** [review]
Thank you! Let's have another @plone/volto-team member review. I just do grammar. 😁

**Comment by sabrina-bongiovanni** [comment]
> Thank you! Let's have another @plone/volto-team member review. I just do grammar. 😁

Thank you Steve 😊

**Review by sneridagh — APPROVED** [review]

**Pull request merged**